<!-- Start ar server -->
<launch>
	<arg name="autostart" default="false"/>
	<arg name="autostart_timeout" default="0.5"/>

	<group ns="ar">
		<param name="task_description" value="$(find tum_ar_window)/config/example.yaml" />
		<param name="autostart" value="$(arg autostart)" />
		<param name="autostart_timeout" value="$(arg autostart_timeout)" />

		<!-- <node name="ar_server" pkg="tum_ar_window" type="ar_server" output="screen" launch-prefix="gdb -ex run - -args" /> -->
		<node name="ar_server" pkg="tum_ar_window" type="ar_server" output="screen" />
	</group>
</launch>